src/frontend/package-lock.json

```diff
@@ -3985,7 +3985,7 @@
     },
     "discontinuous-range": {
       "version": "1.0.0",
-      "resolved": "https://registry.npmjs.org/discontinuous-range/-/discontinuous-range-1.0.0.tgz",
+      "resolved": "https://repo.adeo.no/repository/npm-public/discontinuous-range/-/discontinuous-range-1.0.0.tgz",
```

Comment:
Føler at hvis vi trekker pakker fra repo.adeo.no så går vi et steg tilbake med tanke på åpenhet osv.. Det blir for eksempel verre å bytte byggverktøy til CircleCI eller GitHub Actions hvis vi peker til dependencies bak muren så og si. Er det en grunn til dette?